app/workers/kms_encryption_verification_job.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class KmsEncryptionVerificationJob
+  include Sidekiq::Worker
+  include SentryLogging
+
+  def perform(models = ApplicationRecord.descendants_using_encryption.map(&:name))
+    corrupted_records = models.map(&:constantize).flat_map do |model|
+      attributes = model.lockbox_attributes.keys
+      model.where('encrypted_kms_key IS NOT NULL').flat_map do |record|
+        attributes.flat_map do |attribute|
+          record unless decrypted?(record, attribute)
+        end
+      end
+    end.compact
+    log_corrupted_records(corrupted_records) if corrupted_records.any?
+  end
+
+  private
+
+  def decrypted?(record, attribute)
```

Comment:
I'd name this `can_decrypt?` or `decryptable?` as you're not asking if the data is already decrypted, but if it's capable of being decrypted.